src/Pages/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+   <>
+    <footer>
+        <div className="mainfooter">
+            <div className="firstfoter">
+                <p className='footerbold'>Snap Grid</p>
+                <p>Projectshut is an open source web app that enables users to freely 

```

Comment:
Change this to Snapgrid

Reply:
Also replace that Sentance with : 'SnapGrid serves as a multifaceted online tool that streamlines the process of finding and obtaining images.'